.github/CONTRIBUTING.md

```diff
@@ -61,10 +60,9 @@ Anyone can open a pull request on Primer. You do not need to work at GitHub or b
 
 1. Fork and clone [this repository](https://github.com/primer/primer).
 2. Configure and install the dependencies: `npm install`
-3. Check out the dev branch `git checkout dev`
-3. Create a new branch from dev `git checkout -b my-branch-name`
+3. Create a new branch from master `git checkout -b my-branch-name`
 4. Make your changes and commit them.
-5. Push your branch and open a pull request against `dev`. Add a comment describing your proposed changes and request a review from `@primer/ds-core`.
+5. Push your branch and open a pull request. Add a comment describing your proposed changes and request a review from `@primer/ds-core`.
```

Comment:
Should this be `open a pull request against the latest release branch`?

Reply:
I don't think contributors should have to make that decision. I'd rather they just be open against the default and the core team decide which release branch they fall into.